AMW_angular/io/src/app/deployments/deployments.component.ts

```diff
@@ -364,7 +365,7 @@ export class DeploymentsComponent implements OnInit {
   private pushDownload(prefix: string) {
     this.isLoading = false;
     const docName: string =
-      prefix + '_' + moment().format('YYYY-MM-DD_HHmm').toString() + '.csv';
+      prefix + '_' + datefns.format(new Date(), DATE_FORMAT).toString() + '.csv';
```

Comment:
@srothPuzzle this is not the same format as `DATE_FORMAT`
Can you please change it to `yyyy-MM-dd_HHmm`.